lib/App.js

```diff
@@ -2263,7 +2273,20 @@ class App {
   }
 
   static hasHomeyCompose({ appPath }) {
-    return fs.existsSync(path.join(appPath, '.homeycompose'));
+    const hasComposeFolder = fs.existsSync(path.join(appPath, '.homeycompose'));
+
+    if (
+      hasComposeFolder &&
+      !this.__warnedAboutMissingHomeyComposeManifest &&
+      !fs.existsSync(path.join(appPath, '.homeycompose/app.json'))
+    ) {
+      Log(colors.yellow('⚠ Warning: Could not find a Homey Compose app.json manifest!'));
+      Log(colors.yellow('Using the generated app.json in the root of your app is supported for now,'));
+      Log(colors.yellow('but it is recommended to move your manifest to the .homeycompose/ folder.'));
```

Comment:
Are these separate log lines needed?

Reply:
yea alternatively I could break the lines with `+` and end them with `\n` [like this](https://github.com/facebook/react/blob/48740429b4a74e984193e4e2d364d461e4fdc3be/packages/react-reconciler/src/ReactFiberWorkLoop.new.js#L2882-L2897)